demisto_sdk/commands/unify/tests/yml_unifier_test.py

```diff
@@ -1040,3 +1040,98 @@ def test_add_contributors_support(tmp_path):
         contributor_url='',
     )
     assert unified_yml["display"] == 'Test Integration (Partner Contribution)'
+
+
+def test_add_test_section(tmp_path):
+    '''
+        Given:
+            - an Integration to unify
+
+        When:
+            - the --test flag is True
+
+        Then:
+            - Add a "- Test" to the name/display/id of the integration if the yml exsits.
+    '''
+    unifier = IntegrationScriptUnifier(str(tmp_path))
+    unified_yml = {
+        'display': 'Integration display',
+        'commonfields': {'id': 'Integration id'},
+        'name': 'Integration name'
+    }
+    unified = unifier.add_test_section(unified_yml)
+    assert unified.get('display') == 'Integration display - Test'
+    assert unified.get('name') == 'Integration name - Test'
+    assert unified.get('commonfields').get('id') == 'Integration id - Test'
+
+
+def test_empty_yml(tmp_path):
+    """
+    Given:
+        - An empty unified yml
+
+    When:
+        - calling the add_test_section when using the -t flag
+
+    Then:
+        - Check that the function will not raise any errors.
+    """
+    unifier = IntegrationScriptUnifier(str(tmp_path))
+    unifier.add_test_section({})
+
+
+@pytest.mark.parametrize("flag", [True, False])
+def test_add_test_section_flag_integration(repo, flag):
```

Comment:
this test should be in the `integration_tests` folder

Reply:
Done